### ISSUE TITLE
Add support for laminas/laminas-diactoros v3

### ISSUE THREAD

**Issue by InvisibleSmiley** [trinet-at/mezzio-test#9]
Should be as simple as adding support in composer.json:
https://github.com/laminas/laminas-diactoros/releases/tag/3.0.0

**Comment by rieschl** [comment]
Yes, this package needs some attention. I'm currently quite busy, but when I work on one of the projects this package is used, I'll update it

**Comment by rieschl** [comment]
I just released version [`1.2.0`](https://github.com/trinet-at/mezzio-test/releases/tag/1.2.0) containing support for diactoros v3.
